dotiw.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'i18n'
   s.add_dependency 'activesupport'
 
+  s.add_development_dependency 'actionpack', '>= 4'
+  s.add_development_dependency 'actionview', '>= 4'
```

Comment:
It took me a bit to figure out how to run the specs that depend on `if defined?(ActionView)`. I realized that I needed to install the `actionpack` and `actionview` gems. I decided that including these gems in the gemspec as development dependencies made sense.

Reply:
We don't want that because the library does not require these. Use a gemfile in `gemfiles`, e.g.

```
export BUNDLER_GEMFILE=gemfiles/rails_4.gemfile
bundle install
rake
```

Reply:
See https://github.com/radar/distance_of_time_in_words/blob/master/.github/workflows/ruby.yml#L39 for how we run the combination of things.

Reply:
Since I added those as development dependencies, I thought those gems won't be a requirement for someone that wants to use the gem, only a requirement for someone who wants to work/develop on the gem. 

https://guides.rubygems.org/specification-reference/#add_development_dependency

Is that understanding not correct?

Reply:
That's correct, but it's not actually a development dependency - it's only one when you want to test Rails integration. It will prevent development without Rails.

Reply:
Ok, I will take a look at that. I looked for a section in the README that explained how to develop on this gem and run the specs, but finding none I guess I hacked out a solution that worked for me. You might want to add such a section for future contributors.

Reply:
Removed those development dependencies from the gemspec: https://github.com/radar/distance_of_time_in_words/pull/111/commits/1bd2eb8123006e192029224c3f11e3c233963980

Reply:
👍